doc/user/content/_index.md

```diff
@@ -42,8 +47,8 @@ data freshness**.
 -   [Time-windowed computation](/sql/patterns/temporal-filters/)
     {{</ linkbox >}}
     {{< linkbox icon="book" title="Reference" >}}
--   [`CREATE SOURCE`](/sql/create-source/)
```

Comment:
Since we want to have people use the console ... swapped it out for Create index.

Reply:
We should keep it around until we finish the work to thoroughly support source creation!